infrastructure/file_metadata_service.go

```diff
@@ -46,6 +45,10 @@ func (ms fileMetadataService) GetPublicKey() (string, error) {
 }
 
 func (ms fileMetadataService) GetInstanceID() (string, error) {
+	if ms.metaDataFilePath == "" {
+		return "", nil
+	}
+
```

Comment:
`GetInstanceID` returns error if something is wrong, the idea is that someone who calls this function is responsible for processing this case. In the same time, missing `metaDataFilePath` is an error and it will be processed by `ReadFile`, so I guess the condition should be removed.